src/core/collect/collector.sol

```diff
@@ -14,17 +14,9 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity >=0.4.24;
```

Comment:
We plan to change the version. Please in a separated PR with adjusted travis script

Reply:
I've updated travis to support 0.5.12, we can fix all the other versions in 0.5.12 later. I agree.